ref: update exp init --type value (#3657)
iterative/dvc.org: content/docs/command-reference/exp/init.md

  `dvc exp run`).

- `--type` - selects the type of the stage to create. Currently it provides two

>   alternatives: `checkpoint` and `default` (no need to specify this one).
> 

Author: ```suggestion
  alternatives: `checkpoint` (supports logging
  [checkpoints](/doc/command-reference/exp/run#checkpoints) during model
  training) and `default` (no need to specify this).
```

- `--type` - selects the type of the stage to create. Currently it provides two
  alternatives: `checkpoint` and `default` (no need to specify this one).

> 

Author: ```suggestion
  alternatives: `checkpoint` (supports logging
  [checkpoints](/doc/command-reference/exp/run#checkpoints) during model
  training) and `default` (no need to specify this).
```

>   `checkpoint` supports logging
>   [checkpoints](/doc/command-reference/exp/run#checkpoints) during the training

Author: ```suggestion
```

>   of models.

Author: ```suggestion
```

- `--live` - set the path to the directory where the metrics and plots
  [produced by DVCLive](https://dvc.org/doc/dvclive/dvclive-with-dvc#outputs)

>   will be found. If `--live` is provided, it will override the default values
>   for `--metrics` and `--plots`.

Author: ```suggestion
  will be found. Overrides the default values for `--metrics` and `--plots`.
```
